presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1199,15 +1200,6 @@ public void testArrayTimestampWithTimeZone(boolean insertWithPresto)
         tests.execute(getQueryRunner(), sessionWithArrayAsArray(), dataSetup);
     }
 
-    @DataProvider
-    public Object[][] testTimestampWithTimeZoneDataProvider()
```

Comment:
The cmt title is

> Rename misleading testTimestampWithTimeZoneDataProvider

it's removal not "rename"

"misleading" is misleading. It was very nicely called "<test-name>DataProvider", i do not find anything misleading in that and next time would call it the same.

Reply:
Changed to "Use generic trueFalse data provider for timestamp tests"